core/src/consensus.rs

```diff
@@ -344,19 +344,19 @@ impl Tower {
         self.lockouts.root_slot
     }
 
-    // a slot is not recent if it's older than the newest vote we have
```

Comment:
Inverted the description for more simpler definition. Also, old description should be _older than **or equal to**_...